lib/membrane_rtmp_plugin/rtmp/source/client_handler_for_source.ex

```diff
@@ -1,10 +1,10 @@
-defmodule Membrane.RTMP.Source.ClientHandler do
+defmodule Membrane.RTMP.Source.ClientHandlerForSource do
```

Comment:
I understand it's to avoid confusion with the behaviour, but I'd leave it as is, just never alias it. Alternatively let's call it `ClientHandlerImpl`

Reply:
I'm up to the second option ;)